src/code/z_actor.c

```diff
@@ -1894,7 +1897,9 @@ s32 func_8002F9EC(PlayState* play, Actor* actor, CollisionPoly* poly, s32 bgId,
     return false;
 }
 
-// Local data used for Farore's Wind light (stored in BSS, possibly a struct?)
+#pragma increment_block_number 22
+
+// Local data used for Farore's Wind light (stored in BSS)
 LightInfo D_8015BC00;
 LightNode* D_8015BC10;
```

Comment:
I removed the comment because retail BSS ordering shows these are actually separate variables, there's other variables in between `D_8015BC10` and `D_8015BC14`, and `D_8015BC14` and `D_8015BC18` now (they're "interleaved" with `sCurCeilingPoly` and `sCurCeilingBgId`)